github/libp2p.yml

```diff
@@ -5472,6 +5472,7 @@ teams:
     members:
       maintainer:
         - AgeManning
+        - elenaf9
```

Comment:
you have to remove Elena's username from the `member` list @jxs

Reply:
This is correct. The other issue that I mentioned - the one with sync - is already resolved.

Reply:
thanks! Updated